### PR TITLE
PARQUET-512: Add Google benchmark for performance testing

### PR DESCRIPTION
**Pull request by xhochy**
Based on @emkornfield 's work in https://github.com/apache/arrow/pull/29

### CODE REVIEW

File: build-support/run-test.sh [hunk 1]
```diff
@@ -51,40 +56,57 @@ set -o pipefail
...
-else
-  pipe_cmd=cat
-fi
+pipe_cmd=cat
 
 # Allow for collecting core dumps.
-PARQUET_TEST_ULIMIT_CORE=${PARQUET_TEST_ULIMIT_CORE:-0}
-ulimit -c $PARQUET_TEST_ULIMIT_CORE
+ARROW_TEST_ULIMIT_CORE=${ARROW_TEST_ULIMIT_CORE:-0}
+ulimit -c $ARROW_TEST_ULIMIT_CORE
```
Comment (wesm) [inline build-support/run-test.sh]: Buglet from Arrow porting

**Comment by wesm** [comment]
Will review the benchmark code a bit later. Can you update the Travis CI config to only run the unit tests?

**Comment by xhochy** [comment]
Benchmarks are now build but not run.

File: .travis.yml [hunk 2]
```diff
@@ -28,7 +28,7 @@ matrix:
     os: linux
     before_script:
     - source $TRAVIS_BUILD_DIR/ci/before_script_travis.sh
-    - cmake -DCMAKE_CXX_FLAGS="-Werror" -DPARQUET_TEST_MEMCHECK=ON -DPARQUET_GENERATE_COVERAGE=1 $TRAVIS_BUILD_DIR
+    - cmake -DCMAKE_CXX_FLAGS="-Werror" -DPARQUET_TEST_MEMCHECK=ON -DARROW_BUILD_BENCHMARKS=ON -DPARQUET_GENERATE_COVERAGE=1 $TRAVIS_BUILD_DIR
```
Comment (emkornfield) [inline .travis.yml]: copy and paste bug?

Comment (xhochy) [inline .travis.yml]: Yes 😢

**Comment by wesm** [comment]
+1, thank you

**Pull request closed without merging**